run.mac.sh

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+osname=$(uname)
```

Comment:
Yes, you can, but can do either run cli over them or use the StochSS ui over them but not both at the same time